tests/test_transforms_copy_into.py

```diff
@@ -0,0 +1,30 @@
+import snowflake.connector.cursor
+import sqlglot
+
+from fakesnow.transforms.copy_into import copy_into
+
+example1 = """
+COPY INTO schema_.table_ (a, b)
+FROM "s3://bucket/"
```

Comment:
A real Snowflake instance will throw a SQL compilation error if double quotes are used
```suggestion
FROM 's3://bucket/'
```

Reply:
Updated in 84a30fa